lua/ui/dialogs/options.lua

```
@@ -399,89 +433,196 @@ function CreateDialog(over, exitBehavior)
+            end
+        end
+
+        local row = 1
+        local function AddOptionsFromTabData(tData)
+            for index, option in tData.items do
+                if isSearching then
+                    local titleStr = LOC(option.title)
+                    -- Note: string.lower does not fully handle Unicode case matching
+                    if not string.find(string.lower(titleStr), string.lower(searchString), 1, true) then
+                        continue
+                    end
```

Luacheck (1.2.0)

[error] expected '=' near 'end'

(E011)